examples/logging/colored-logging-config.yaml

```diff
@@ -0,0 +1,67 @@
+# Example logging configuration with colored output
```

Comment:
rather than create a new config, we could add the coloredFormatter to the existing https://github.com/SolaceLabs/solace-agent-mesh/blob/main/templates/logging_config_template.yaml
We could consider defaulting to it as well